### ISSUE TITLE
setup mailchimp account and handoff

### ISSUE THREAD

**Issue by ScottDowne** [mozilla/donate.mozilla.org#1409]
Putting this on @simonwex initially because he has credit card access.

We'll need to hand over appropriate credentials, create accounts for people, setup the account with an account name and email that people see when we send them emails (thunderbird-newsletter@mozilla.org or equivalent). Then plug in api keys to staging. I need a key, a list id (we also need to create that list) and the account name. The new account creation process makes that pretty straightforward.

The credentials go into heroku donate production, I can handle that part once we have the account.

cc @gerv @rkent @davidascher @edrushka

Home stretch! Almost there! Thanks for everyone's patience! :zap::zap::zap::cake:

**Comment by rkent** [comment]
I have already setup the mailchimp account with the username thunderbird-accounts@mozilla.org  I can send the login credentials to whoever needs to test or configure this.

**Comment by ScottDowne** [comment]
Oh cool, I thought you setup a test account, but that works too.

**Comment by rkent** [comment]
So who do I send the credentials to?

**Comment by ScottDowne** [comment]
Someone will request it in this ticket, probably @simonwex

**Comment by simonwex** [comment]
Hey @rkent, could you send the details to @cadecairos? Best bet would be to PGP and send to cade@mozillafoundation.org. You can find his pub key here: https://keybase.io/cadecairos

**Comment by edrushka** [comment]
Cade got the info. We're meeting tomorrow with Scott to wrap this piece up.

**Comment by cadecairos** [comment]
I've added the monthly plan to the mailchimp account. 

@wex @rkent is the idea here for MoFo to take control of the Mailchimp account?

I've added myself and scott to be managers on it.

**Comment by rkent** [comment]
No, I thought that the whole idea is that MoFo did not want to bother with the Mail Chimp account, plus it would be best to not go down the path of trying to get Thunderbird volunteers to have credentials for Firefox-related MoFo accounts.So I do not expect "MoFo to take control of the Mailchimp account". The account will be managed by the Thunderbird Council.

I don't really know who @cadecairoes and @ScottDowne are (Mozilla directories are above my pay grade) - could you give me some basic details on them if they should be managers on this Mail Chimp account?

**Comment by cadecairos** [comment]
Scott and I are Mozilla Foundation developers.

Scott's been added to the account so he can troubleshoot issues while we test things out before launching. I've added myself in case somthing billing-wise changes. (although now that I think about it, the foundation email I added might not have access to billing settings)

**Comment by ScottDowne** [comment]
I think manager was just a loaded term. I just needed access to get the required keys so I can plug them into the donate page and have everything connect. While I do this, I'll need to test, which means signing myself up and thus I'll need to see that I did in fact enter the mailing list.

I don't need to have access for the long term.
